hibernate-testing/src/main/java/org/hibernate/testing/junit4/BaseNonConfigCoreFunctionalTestCase.java

```diff
@@ -483,9 +483,12 @@ protected boolean isCleanupTestDataRequired() {
 	protected void cleanupTestData() throws Exception {
 		Session s = openSession();
 		s.beginTransaction();
-		s.createQuery( "delete from java.lang.Object" ).executeUpdate();
-		s.getTransaction().commit();
-		s.close();
+		try {
```

Comment:
Wouldn't we want to call `s.beginTransaction()` inside the try-block and then add a catch-block to determine if a transaction is active and rollback?